src/analyzer_main.cpp

```diff
@@ -559,6 +560,58 @@ struct Analyzer {
             return false;
         }
 
+        //Handling susceptibility
+        if(config.use_susceptibility == true && stats.n_steps % 10 == 0){
+
+            ofstream myfile;
+
+            stringstream ss;
+            ss << "output/change_in_ideology_step_" << stats.n_steps << ".dat";
```

Comment:
I'm fine with anything you decide. Just noting my concern that it could be a lot of output.